### DISCUSSION TITLE
Account Restriction - Unable to Authorize Third-Party Application

### DISCUSSION THREAD

**Discussion by v1nqour** [community/community#60924] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Dear GitHub Support,

I hope this message finds you well. I am writing to seek assistance with an issue I am encountering while trying to log in with my GitHub account on Netlify.

Whenever I attempt to authorize a third-party application, specifically Netlify, I receive an error message stating, "You are marked as spam, and therefore cannot authorize a third-party application." This restriction is preventing me from accessing the services and functionality provided by Netlify.

I want to assure you that I take the guidelines and terms of service on GitHub seriously and have made efforts to comply with all policies. However, I am unaware of any actions or activities that might have led to my account being flagged as spam.

I kindly request your support in investigating and resolving this matter. I would appreciate any information you can provide about the reason for the account restriction and any steps I can take to rectify the situation. Additionally, if there are any specific guidelines or actions I need to follow to regain access to authorize third-party applications, please let me know.

I value my GitHub account and the services it provides, and I would like to continue using it seamlessly with external applications such as Netlify. Your prompt attention and assistance in resolving this issue would be highly appreciated.

Thank you for your time and consideration. I look forward to your response and guidance on resolving the account restriction.

Sincerely,

**Comment by LiteBrite82** [comment, marked as answer]
Hi @v1nqour and @Team-Bota-Apex 

Im so sorry I missed your posts!

This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

That being said; to resolve the issue, you need to enable 2FA in your account and open a support ticket to get the flag removed. [see discussion](https://github.com/orgs/community/discussions/67196)

@nuhuwambali made a [good video](https://www.youtube.com/watch?v=bUdCUhfLdtc&t=63s) that walks you through the process.

**Comment by NuhuWambali** [comment]
the video link is this one please share to the community
https://www.youtube.com/watch?v=bUdCUhfLdtc&t=63s

**Reply by LiteBrite82** [reply]
Thanks @NuhuWambali - I totally forgot to include the link in my post!

**Comment by NuhuWambali** [comment]
No problem, I understand.
